test/es6/rlexe.xml

```diff
@@ -1421,6 +1421,12 @@
 </test>
 <test>
   <default>
+    <files>bug_issue_3076.js</files>
+    <tags>BugFix</tags>
```

Comment:
No, but it is better to use forcedeferparse. Updated.

---
In reply to: [120246928](https://github.com/Microsoft/ChakraCore/pull/3111#discussion_r120246928) [](ancestors = 120246928)